src/typings/index.d.ts

```diff
@@ -0,0 +1,101 @@
+export type StakedSNXResponse = {
```

Comment:
d.ts file can behave a little wierd. I would only use that if you're adding type to an untyped package. Change this to index.ts or types.ts

Reply:
Sure